Testing/functional/tests/cypress/integration/ui/user/profile.js

```diff
@@ -4,7 +4,6 @@ const HDID = "P6FFO433A5WPMVTGM7T4ZVWBKCSVNAYGTWTU3J2LWMGUMERKI72A";
 
 describe("User Profile", () => {
     beforeEach(() => {
-        cy.enableModules([]);
```

Comment:
You might want to replace this with `cy.configureSettings({});` just to ensure everything unimportant is disabled.